addons/point_of_sale/static/src/app/screens/receipt_screen/receipt_screen.js

```diff
@@ -33,14 +33,22 @@ export class ReceiptScreen extends Component {
             this.currentOrder.uiState.locked = true;
             this.pos.sendOrderInPreparation(order);
         });
+        this.emails = new Set();
     }
 
     _addNewOrder() {
         this.pos.add_new_order();
     }
     actionSendReceipt() {
         if (this.state.mode === "email" && this.isValidEmail(this.state.input)) {
-            this.sendReceipt.call({ action: "action_send_receipt", name: "Email" });
+            if (this.emails.has(this.state.input)) {
```

Comment:
![image](https://github.com/user-attachments/assets/f91a4950-35f4-4a5d-a34a-c9f8b6f0c68e)
what about something like this ? We already have stored the information about the sent emails, so there's no need to store it again here